portage-stubs/__init__.pyi

```diff
@@ -31,7 +33,7 @@ class dbapi:
     ) -> tuple[None, Literal[0]] | tuple[str, str] | tuple[str, None]:
         ...
 
-    def match(self, mydep: str, use_cache: Literal[0, 1] = ...) -> list[str] | str:
+    def match(self, mydep: str, use_cache: Literal[0, 1] = 1) -> list[str] | str:
```

Comment:
Should be ... not the literal value.

Reply:
https://github.com/gentoo/portage/blob/master/lib/portage/dbapi/porttree.py#L1444-L1445


I have seen that now they use that value by default
that's why I had updated the typing